TTKDTrainingApp/src/components/VideoWithControls/VideoWithControls.js

```diff
@@ -32,6 +32,11 @@ const VideoWithControls = props => {
   const [duration, setDuration] = useState(0);
   const [rate, setRate] = useState(DEFAULT_SPEED);
   const [loading, setLoading] = useState(true);
+  const [showControls, setShowControls] = useState(false);
+
+  const [videoHeight, setVideoHeight] = useState(0);
+  const [videoRatio, setVideoRatio] = useState(16 / 9);
```

Comment:
for playing the video do you think when we do landscape mode the video would fill up the entire screen or this might be later thing for paul to decide honestly

Reply:
That'll be included in another PR in the future, and we can double-check with Paul to see if it's a change he wants to make